0.1.Copy data from Excel codes.R

```diff
@@ -1,79 +1,75 @@
-rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
-gc() #free up memrory and report the memory usage.
+#rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
+#gc() #free up memrory and report the memory usage.
 memory.limit(size = 80000000)
 
 library(xlsx)
 library(ggplot2)
 library(ggpmisc)
 library(readxl)
+library(dplyr)
+library(forcats)
+library(bayestestR)

```

Comment:
I moved the `bayestestR` loading here because all the libraries should be loaded in one part of the script somewhere near the beginning so you don't have to hunt through the script to figure out what libraries you need.